.github/workflows/go.yml

```diff
@@ -0,0 +1,131 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build-and-test:
+    name: Build Go and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        
+        # --prune --unshallow is needed for changelog generation by goreleaser
```

Comment:
I would put this comment on the line that does the fetch: `# needed for goreleased changelog generation`

Reply:
👍